RFS/src/main/java/com/rfs/common/FileSystemSnapshotCreator.java

```diff
@@ -9,14 +9,14 @@ public class FileSystemSnapshotCreator extends SnapshotCreator {
     private static final Logger logger = LogManager.getLogger(FileSystemSnapshotCreator.class);
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    private final ConnectionDetails connectionDetails;
+    private final OpenSearchClient client;
```

Comment:
nit: what kind of client is this for - it's to upload source cluster data, source cluster metadata, or RFS metadata?

Reply:
Quite literally, a client to interact w/ OpenSearch as a layer above the RestClient.  So, yes.  We may have another client layer abstraction above it (such as the OpenSearchCmsClient class I have in local changes that uses an OpenSearchClient to interact w/ the Coordinating Metadata Store).

Reply:
That tells me what the class is.  I was asking about what this particular instance was being used for within this class.  I was trying to understand how this class worked.

Reply:
Well, it's used to take a snapshot from source cluster to the local filesytem.  Should be self-evident from the class/method names?  If not, open to suggestions for ways to make it more obvious.